src/WeatherDisplay.js

```diff
@@ -0,0 +1,52 @@
+import "./forecastStyle.css";
+import weatherData from "./WeatherData.js"
+
+//this is a class used to display one weatherData element at a time
+//its objects are supposed to be made and used in ForecastDisplay objects
+
+export default class weatherDisplay {
+
+    constructor(containerId){
+        this.container=document.getElementById(containerId);
+        this.container.className="weatherContainer"
```

Comment:
To "niebezpieczna" konstrukcja - zwłaszcza, kiedy Twój komponent siada "na" wskazanym elemencie a nie "wewnątrz" niego (co samo w sobie zwykle nie jest najlepszym pomysłem).

Lepiej zazwyczaj użyć
```
classList.add()
```
w ten sposób nie usunie się w nieplanowy sposób klas, którymi wcale nie chcieliśmy się zajmować.